apps/ensindexer/src/lib/lib-config.ts

```diff
@@ -43,3 +43,23 @@ export function getRpcConfigsFromEnv(): Record<number, RpcConfigEnvironment> {
 
   return rpcConfigs;
 }
+
+export function prettyPrintConfig(config: ENSIndexerConfig) {
+  return JSON.stringify(
+    {
+      ...config,
+      databaseUrl: "*******",
+      rpcConfigs: Object.fromEntries(
+        Object.entries(config.rpcConfigs).map(([chainId, rpcConfig]) => [
+          chainId,
+          {
+            ...rpcConfig,
+            url: "*******",
+          },
+        ]),
+      ),
+    } as ENSIndexerConfig,
+    null,
+    2,
+  );
```

Comment:
Feel like a nice candidate for the `toPublic` method we spoke about some other time: https://github.com/namehash/ensnode/issues/521#issuecomment-2802662311

I'll take a note to come back to this function once ENSNode SDK comes to life 🚀